beacon-chain/db/slasherkv/slasher.go

```diff
@@ -361,18 +361,17 @@ func (s *Store) HighestAttestations(
 	history := make([]*slashpb.HighestAttestation, 0, len(encodedIndices))
 	err = s.db.View(func(tx *bolt.Tx) error {
 		signingRootsBkt := tx.Bucket(attestationDataRootsBucket)
-		attRecordsBkt := tx.Bucket(attestationRecordsBucket)
 		for i := 0; i < len(encodedIndices); i++ {
 			c := signingRootsBkt.Cursor()
 			for k, v := c.Last(); k != nil; k, v = c.Prev() {
 				if suffixForAttestationRecordsKey(k, encodedIndices[i]) {
-					encodedAttRecord := attRecordsBkt.Get(v)
+					encodedAttRecord := v[signingRootSize:]
 					if encodedAttRecord == nil {
 						continue
 					}
-					attWrapper, err := decodeAttestationRecord(encodedAttRecord)
-					if err != nil {
-						return err
+					attWrapper, err1 := decodeAttestationRecord(encodedAttRecord)
+					if err1 != nil {
+						return err1
```

Comment:
I'm curious about this change. Why rename err to err1? Was it to satisfy some err / shadowing linter, personal preference, or some other standard we should adopt?

Reply:
Yes. err is returned by View. By naming like this we can avoid shadowing. We should actually use more legible errName. I have changed this to 'decodeErr'